lib/utils.js

```diff
@@ -59,7 +59,40 @@ function init() {
 
   };
 
-  return utils();
+  // some helpers for input "date"
+  utils.mergeInputTime = function mergeInputTime(timeid,dateid) {
+    var value = $(timeid).val();
```

Comment:
don't think we should be using jquery in `utils.js` since this module is used on the server too.  Can we pass the values instead of the dom ids?